### ISSUE TITLE
Simplify loop playing

### ISSUE THREAD

**Issue by Shine753** [helge17/tuxguitar#441]
Using TuxGuitar 1.6.3
Looping some part of the tablature is a bit complicated. It would be great to have just to select some part, and activate a loop button. Si only this part is played and looped. This would be done by extracting the top section of the loop dialog, adn put it on the main top bar.

**Comment by guiv42** [comment]
Hi
Could you please elaborate a bit? In fact this is exactly how I start a loop with 1.6.3:
- select some part
- click on play mode icon
- activate "play looped" option, click OK
- then play

https://github.com/helge17/tuxguitar/assets/129443524/eac19f02-d77b-4075-87d8-77529b01af2e

It's even possible to go faster:
- click in measure X
- shift-click in measure Y
- type F9, click "play looped", OK
- then press space bar

I really don't know how to make it simpler

**Comment by Shine753** [comment]
I suggest something like this, that should be more user friendly : 
![Proposal](https://github.com/helge17/tuxguitar/assets/37425160/135a8676-a0a8-4992-bdc4-e0a505f44bb0)

You just select the part you want to play (or select nothing that mean you want to play from where you are to the end), then click play.
That way, the play mode is easier to access, and more contextual (with less visual options that stack up in a dialog box).

**Comment by guiv42** [comment]
OK, that would mean globally integrating the player mode dialog in main toolbar.
Note that other dialogs are legitimate candidates for an integration in the main toolbar (e.g. transport dialog Ctrl+t).

Not a simple evolution, this requires a global rework of the toolbar: current implementation of toolbar can only host buttons easily (not other widgets like drop-down lists, ...)

**Comment by Shine753** [comment]
I see. Changing the framework used for the toolbar would allow you to unlock great UI features.
But as an intermediate option, you could use the existing code by :
- reworking the play options dialog so it displays 3 distinct parts, one for each play mode (and remove the bottom part that is useless, see my previous suggestion)
- going with the following kind of design

![image](https://github.com/helge17/tuxguitar/assets/37425160/38d67bd2-59d1-4d88-ab5a-a91ebdcaf8ae)

That way you always know in which mode you are, and are able to cycle through modes with a shortcut or a click.
The only thing is that you don't know what are the defined speeds, and can't adjust them easily. But that could come in a second time when the rework of the toolbar using a new framework is done.

What do you think about this ?

**Comment by guiv42** [comment]
> Changing the framework used for the toolbar would allow you to unlock great UI features.

I fully agree. It should take time (I already tried last year, and failed...), but this seems to me the cleanest solution
